### ISSUE TITLE
[FEAT] Hide solo tab

### ISSUE THREAD

**Issue by Faruolo2** [AdamXweb/WhiteSurFirefoxThemeMacOS#52]
Could you please make the solo tab to hide to make it look more like safari?

the code below kinda works:

#tabbrowser-tabs tab[first-visible-tab="true"][last-visible-tab="true"]:not([pinned]),
#tabbrowser-tabs tab[first-visible-tab="true"][last-visible-tab="true"]:not([pinned]) ~ toolbarbutton {
  visibility: collapse;
}

but the tab area remains black.

**Comment by AdamXweb** [comment]
This may be an issue, as UI of safari has new tab button in top menu bar area, however in firefox shows in the tabs bar.
May look into relocating it to the tabsbar, which may provide the opportunity for this

**Comment by maxwolv** [comment]
This user seems to have found workable CSS with the new proton FireFox to hide a single tab, maybe you can integrate into white Sur?

https://github.com/MrOtherGuy/firefox-csshacks/blob/master/chrome/hide_tabs_with_one_tab.css

**Comment by maxwolv** [comment]
As for the new tab button, given this is a pretty niche theme that needs to be installed via bash script, I bet most users are using command + T :)

**Comment by maxwolv** [comment]
> This may be an issue, as UI of safari has new tab button in top menu bar area, however in firefox shows in the tabs bar. May look into relocating it to the tabsbar, which may provide the opportunity for this

You can manually drag the new tab button to the top menu bar

**Comment by AdamXweb** [comment]
Good pickup! I only learnt last month that you could customise the toolbar and drag it up for that clean look

**Comment by AdamXweb** [comment]
I'll look into integrating something. It'd most likely be an install option for users looking to customise!

**Comment by luan** [comment]
This worked for me after some trial and error:

```css
#tabbrowser-tabs, #tabbrowser-tabs arrowscrollbox {
  height: auto !important;
  padding: 0 !important;
}

#TabsToolbar {
  min-height: 0 !important;
}

#tabbrowser-tabs tab[first-visible-tab="true"][last-visible-tab="true"]:not([pinned]),
#tabbrowser-tabs tab[first-visible-tab="true"][last-visible-tab="true"]:not([pinned]) ~ #titlebar {
  display: none;
}

```

**Comment by NyaomiDEV** [comment]
> This worked for me after some trial and error:

Yeah, but it isn't really animated and it looks pretty janky :(

**Comment by AdamXweb** [comment]
Thanks @luan for the code! I am satisfied as Safari has the same look when going to solo tab view.
Will be available in v1.5.1 with the install flag `-s`
<img width="1328" alt="Screen Shot 2023-01-25 at 5 54 23 pm" src="https://user-images.githubusercontent.com/6800453/214499429-36413e29-0167-400c-969f-46d2d7082559.png">
